step.yml

```diff
@@ -50,11 +71,15 @@ inputs:
       title: "Deploy directory or file path"
       description: |
         Specify the directory or file path which will be deployed.
+        
         If the specified path is a directory, then every file
-        in the specified directory will be deployed, sub-directories, however,
-        won't be uploaded. To upload the directory's content
-        recursively, you should use the *Compress* option
-        which will compress the whole directory, with
+        in the specified directory will be deployed. If, however,
```

Comment:
`If, however, the specified path is a sub-directory, it won't be uploaded. `

The specified path will always be a directory (never a subdirectory because it is relative). Every file in the specified directory will be uploaded and **no** files in the subdirectories found in the specified directory